src/components/SkillEditor/SkillEditor.js

```diff
@@ -148,82 +189,227 @@ export default class Container extends React.Component {
+                else{
+                    self.setState({
+                        loading:false
+                    });
                     notification.open({
                         message: 'Error Processing your Request',
-                        description: 'Error in processing the request. Please try with some other skill',
-                        icon: <Icon type="close-circle" style={{ color: '#f44336' }} />,
+                        description: String(data.message),
+                        icon: <Icon type="close-circle" style={{ color: '#f44336' }} />
                     });
                 }
+            })
+            .fail(function (jqXHR, textStatus) {
+                self.setState({
+                    loading:false
+                });
+                notification.open({
+                    message: 'Error Processing your Request',
+                    description: String(textStatus),
+                    icon: <Icon type="close-circle" style={{ color: '#f44336' }} />
+                })
             });
     }
 
-
     render() {
         const style = {
             width: "100%",
             padding: "10px"
-        };
+        }
         return (
           <div>
             <StaticAppBar {...this.props} />
             <div style={styles.home}>
                 <Paper style={style} zDepth={1}>
                   <div>Currently Editing : <h3>{url}</h3></div>
+                    <div style={styles.center}>
+                        <div style={styles.dropdownDiv}>
+                            <SelectField
+                                floatingLabelText="Category"
+                                style={{width:'160px',marginLeft:10,marginRight:10}}
+                                value={this.state.groupValue}
+                                onChange={this.handleGroupChange}
+                            >
+                                {groups}
+                            </SelectField>
+                            <SelectField
+                                floatingLabelText="Language"
+                                style={{width:'90px',marginLeft:10,marginRight:10}}
+                                value={this.state.languageValue}
+                                onChange={this.handleLanguageChange}
+                            >
+                                {languages}
+                            </SelectField>
+                            <TextField
+                                floatingLabelText="Enter Skill name"
+                                floatingLabelFixed={true}
+                                style={{marginLeft:10,marginRight:10}}
+                                value={this.state.expertValue}
+                                onChange={this.handleExpertChange}
+                            />
+                            { this.state.showImage &&  <img alt="preview" id="target" style={{width:60,height:60,borderRadius:"50%",marginRight:20,border: 0}}src={this.state.image}/> }
```

Comment:
Can you change inline style to an external style by having a css class for this and the corresponding style over there?